src/lib.rs

```diff
@@ -225,22 +225,25 @@ impl Executor {
         // already, then be aggressive: wake all idle threads and spawn one more thread.
         while inner.queue.len() > inner.idle_count * 5 && inner.thread_count < EXECUTOR.thread_limit
         {
-            // The new thread starts in idle state.
```

Comment:
Is there a correctness/performance reason why the notification/adjustment happened here?

Reply:
Ah, I see it now. I moved the count bump to before the thread spawning.